doc(release) Release notes collect 24.04.8 / gorgone 24.04.5 (#3912)
centreon/centreon-documentation: versioned_docs/version-24.04/releases/centreon-os.mdx

## Centreon Collect

### 24.04.8

Release date: `December 18, 2024`

<details>
  <summary>Bug fixes</summary>

> - [Engine] Fixed errors in RRD logs that appeared following the upgrade to the new log engine in collect.
> - [Engine] Fixed an issue on retention.dat deletion after an upgrade.
> - [Engine] Fixed an issue where a segmentation fault could occur when host relations were modified.

Reviewer: ```suggestion
- [Engine] Fixed an issue where retention.dat files were deleted after an upgrade.
- [Engine] Fixed an issue where a segmentation fault could occur when host relations were modified.
- [Engine] Fixed errors in RRD logs that appeared following the upgrade to the new log engine in Collect.
```

## Centreon Gorgone

### 24.04.5

Release date: `December 18, 2024`

<details>
  <summary>Bug fixes</summary>

> - [Gorgone] Fixed a bug in gorgone where if the plateform have thousands of hosts, pushing the configuration failed and gorgone connection was severed.

Reviewer: ```suggestion
- [Gorgone] Fixed a bug in Gorgone affecting platforms with thousands of hosts, where deploying the configuration failed and the connection with Gorgone was lost.
```
